app/assets/stylesheets/calendars/index.scss

```diff
@@ -0,0 +1,38 @@
+.yearnav {
+  .left {
+    left: 0;
+  }
+  .center {
+    text-align: center;
+  }
+  .right {
+    right: 0;
+  }
+  select {
+    margin-top: 10px;
+  }
+} 
+
+table.column {
+  margin: 0 10px;
+  margin-bottom: 1.4em;
+}
+
+.calendar {
+  display:inline-table;
+
+  caption {
+    text-align: center;
+  }
+  th {
+    text-align: center;
+  }
+  td {
+    font-size:12px;
```

Comment:
Can we get

1. A space after `:`
2. `line-height: 30px` so it's not quite so vertically smushed

Reply:
Ah, that looks much better, thanks! 
After changing the line height I noticed the columns were also horizontally smushed around the Thu,Fri,Sat columns. I've made them even widths by adding `table-layout: fixed;`